elastic/kibana
# [Ingest Manager] Log a warning if registryUrl is set in non gold

## Summary

Currently user need to have a gold licence to set a custom EPM registry url, otherwise it will silently use the default one.

This PR add a warning in the log message if customUrl is set and no gold licence are active.

## How to test?

Run ES with a basic licence, and set the customUrl you should see a warning message.
